fetch_data_from_database.py

```diff
@@ -0,0 +1,51 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
+from Layout_table import Base,table_structure
+import os
+#connect to database
+def database_connect():
+    path = 'mysql+mysqlconnector://{}:{}@localhost:{}/{}'.format(os.getenv('User_name'),os.getenv('Password'),os.getenv('Port'),os.getenv('Db_name'))
+    engine = create_engine(path)  
+    return engine
+
+
+#create a session
+def Session_create(sess_engine):
+	sess_maker = sessionmaker()
+	sess_maker.configure(bind=sess_engine)
+	sess = sess_maker()
+	return sess
+
+#Fetch the data from database
+    
+def fetch(sess,locate):
+    list_data = locate.split()
+    final_str = str()
+    for i in range(len(list_data)):
+        list_data[i] =list_data[i].capitalize()
+        final_str+=list_data[i]+' '
+    final_str+='%'
+    result = sess.query(table_structure).filter(table_structure.name_of_loaction.like(final_str)).all()
+    return result[0].__dict__
```

Comment:
I like the idea of keeping the SQLAlchemy related objects inside this function and returning a dict instead